src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -203,7 +225,11 @@ private synchronized void updateNavigatorField()
     private void initTasks()
     {
         this.tasks.addTask(0, new EntityAISwimming(this));
-        this.tasks.addTask(1, new EntityAICitizenAvoidEntity(this, EntityMob.class, 8.0F, 0.6D, 1.6D));
+
+        if (this.getColonyJob() == null || !this.getColonyJob().getName().equals("com.minecolonies.job.Guard"))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "com.minecolonies.job.Guard" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1132)